docs/pages/example/point-from-geocoder-result.html

```diff
@@ -13,7 +13,12 @@
     container: 'map',
     style: 'mapbox://styles/mapbox/streets-v9',
     center: [-79.4512, 43.6568],
-    zoom: 13
+    zoom: 13,
+    addToMap: true,
+    markerOptions: {
+        color: 'orange',
+        draggable: true
```

Comment:
What's the reasoning behind `draggable: true`? I get it shows options you can pass in, but I'm not sure if this makes sense as a default.

Reply:
Entirely as an example of being able to pass in options. Perhaps just the `color` option would suffice as an illustrative example.